racktables/find_pingable_machines.sh

```diff
@@ -1,4 +1,4 @@
 #!/bin/bash
 python3 get_unused_machines.py | while read fqdn; do
     ping -c1 -W1 "${fqdn}" &> /dev/null && echo "${fqdn} up"
-done
+done && exit 1 || exit 0
```

Comment:
If you want to turn around the exit code you should be able to do that with

```
! python3 … ; do
…
done
```

Reply:
your suggestion looks good. However while testing I realized that without a temporary variable to store the state, I will most likely not archive what I want with this change (fail if _any_ host is reachable, not just fail if the last one is reachable)

Reply:
ok, nevermind. Just keep as is then I guess

Reply:
got pretty ugly now. If you know of a better way please let me know :)